sbc_compassion/models/import_letters_history.py

```diff
@@ -348,15 +354,17 @@ def _find_qrcode(self, img, MULTIPAGE, file_,
         if qrcode is None:
             # rotate image
             img = img[::-1, ::-1]
-            # save it for zxing
-            cv2.imwrite(file_png, img)
```

Comment:
If QR code is not detected in the bottom-right corner, it makes no sense to save the rotated image